### PR TITLE
CVE-2020-13379: Upgrade Grafana image to 6.7.4

### LINKED ISSUE #2600: CVE-2020-13379: Incorrect access control vulnerability in Grafana
**Component**:

'build', 'grafana'

**Why this is needed**:

We currently ship the following Grafana versions which are vulnerable:
For 2.4 => 6.4.2
For 2.5 =>  6.7.1

See: https://grafana.com/blog/2020/06/03/grafana-6.7.4-and-7.0.2-released-with-important-security-fix/?utm_source=grafana_news&utm_medium=rss

**What should be done**:

Our Grafana versions have a direct binding to the Prometheus-operator chart we ship. Directly bumping the Grafana version could raise compatibility issues.

**Implementation proposal** (strongly recommended):
TBD

**Test plan**:

### PR DESCRIPTION
**Pull request by gdemonet**
**Component**: charts, build

**Context**: Security fix (see #2600)

**Summary**:

- bumped the image tag to 6.7.4 in both the charts and the buildchain

**Acceptance criteria**: 

Grafana is still working as expected, piecharts being available.

---

Closes: #2600

### CODE REVIEW

**Comment by gdemonet** [comment]
Small update: `grafana:6.7.4` image doesn't include the `piechart-plugin` installed out-of-the box. We will keep the custom-built image for 2.4.4 release, but aim at changing this behaviour in the future (notably, to make it easier for users/Solutions to install additional plugins).

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## Release 2.4.4 (in development)
+
+### Bug fixes
+- [#2600](https://github.com/scality/metalk8s/issues/2600) - Upgrade Grafana to
+6.7.4 to handle CVE-2020-13379
+(PR [#2605](https://github.com/scality/metalk8s/issues/2605))
+
```
Comment (gdemonet) [inline CHANGELOG.md:9]: Sorry @Ebaneck, might conflict on your PR, maybe you want to cherry-pick the relevant commit, or rebase on my branch

Comment (Ebaneck) [reply CHANGELOG.md:9]: ok

**Review by Ebaneck — APPROVED** [review]
LGTM

**Pull request merged**